src/Common/tests/System/PlatformDetection.cs

```diff
@@ -43,6 +43,33 @@ public static partial class PlatformDetection
 
         public static int WindowsVersion => GetWindowsVersion();
 
+        public static bool IsAtLeast462()
+        {
+            if (!IsFullFramework)
+            {
+                return true;
```

Comment:
Is Windows10_1073 returns false on !Windows.  I think this should return false on !netfx.  The local property would then be more declarative:

```C#
private static bool EphemeralKeysAreExportable { get; } = !PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer;
```

Which also makes me think that this should have Netfx somewhere in the name.